store/etcdv3/go.mod

```diff
@@ -3,15 +3,8 @@ module github.com/jasonjoo2010/goschedule/store/etcdv3
 go 1.14
 
 require (
-	go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200925060232-add86bbd1a7a
-	github.com/coreos/go-semver v0.3.0 // indirect
-	github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf // indirect
-	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
-	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/jasonjoo2010/goschedule v1.1.0
 	github.com/labstack/gommon v0.3.0
-	github.com/sirupsen/logrus v1.6.0
 	github.com/stretchr/testify v1.7.0
-	go.uber.org/zap v1.16.0 // indirect
-	google.golang.org/grpc v1.29.1 // indirect
+	go.etcd.io/etcd/client/v3 v3.5.0
```

Comment:
Is it necessary to upgrade the dependency for etcdv2 protocol? Or the new upgrading only supports etcdv3

Reply:
I'm not sure